e2e/components/CreateFlows/CreateFullPage-test.avt.e2e.js

```diff
@@ -25,4 +27,40 @@ test.describe('CreateFullPage @avt', () => {
       'CreateFullPage @avt-default-state'
     );
   });
+
+  test('@avt-error-disabled-state', async ({ page }) => {
+    await visitStory(page, {
+      component: 'CreateFullPage',
+      id: 'ibm-products-patterns-create-flows-createfullpage--create-full-page',
+      globals: {
+        carbonTheme: 'white',
+      },
+    });
+    const inputElement = await page.locator('#test-1');
+    await inputElement.click();
+    await page.mouse.click(0, 0);
+    const dataInvalid = await inputElement.getAttribute('data-invalid');
+    const nextButtonElement = page.locator('button:has-text("Next")');
+    const backButtonElement = page.locator('button:has-text("Back")');
+    const isNextDisabled = await nextButtonElement.evaluate((button) =>
+      button.hasAttribute('disabled')
+    );
+    const isBackDisabled = await backButtonElement.evaluate((button) =>
+      button.hasAttribute('disabled')
+    );
+    await expect(dataInvalid).toBe('true');
+    await expect(isBackDisabled).toBe(true);
+    await expect(isNextDisabled).toBe(true);
+    await inputElement.click();
+    await inputElement.fill('test');
+    await expect(inputElement).toHaveValue('test');
+    const isNextDisabledNow = await nextButtonElement.evaluate((button) =>
+      button.hasAttribute('disabled')
+    );
+    const isBackDisabledNow = await backButtonElement.evaluate((button) =>
+      button.hasAttribute('disabled')
+    );
+    await expect(isNextDisabledNow).toBe(false);
+    await expect(isBackDisabledNow).toBe(true);
```

Comment:
Can you add this as well at the end
const dataInvalid = await inputElement.getAttribute('data-invalid');
await expect(dataInvalid).toBe('false');